### ISSUE TITLE
Errors on startup

### ISSUE THREAD

**Issue by ptman** [tulir/mautrix-whatsapp#2]
Some errors in the log I get on startup:

```
[Aug 28, 2018 10:55:05] [Database/Portal/ERROR] Database scan failed: sql: Scan error on column index 2: unsupported Scan, storing driver.Value type <nil> into type *string
[Aug 28, 2018 10:55:05] [Database/Portal/WARN] Failed to insert 358<phone-nr-1>@s.whatsapp.net->@user:domain.tld: UNIQUE constraint failed: portal.jid, portal.owner
```
And some more:

```
[Aug 28, 2018 10:55:05] [User/@user:domain.tld/Portal/358<phone-nr-1>@s.whatsapp.net/ERROR] Failed to create portal room: msg=Failed to POST JSON to /_matrix/client/r0/createRoom code=403 wrapped=M_FORBIDDEN: Application service has not registered this user
[Aug 28, 2018 10:55:05] [User/@user:domain.tld/Portal/358<phone-nr-2>-<ts?>@g.us/ERROR] Failed to download media: download failed
[Aug 28, 2018 10:55:06] [User/@user:domain.tld/Portal/358<phone-nr-3>@s.whatsapp.net/ERROR] Failed to download media: download failed
```

**Comment by tulir** [comment]
First error should be fixed by #5

**Comment by ptman** [comment]
I'm not getting the rest of the errors anymore either
